roles/autossl/defaults/main.yml

```diff
@@ -1,2 +1,50 @@
 ---
-# defaults file for autossl
+
+## Deployment
+replicas: 1
+image_name: "quay.io/3scale/autossl"
+image_version: "v1.0.0"
```

Comment:
I think that , in general, a default value for "Image" should be a floating tag, like "latest". What do you think?

Reply:
I agree, going to change it!

Reply:
Fixed https://github.com/3scale/saas-operator/pull/2/commits/ca96925ae20505ea32801ca559db00b51dc452df